tests/e2e/idps_test.go

```diff
@@ -780,11 +780,12 @@ var _ = Describe("TF Test", func() {
 			})
 
 			AfterEach(func() {
-				err := idpService.google.Destroy()
-				Expect(err).ToNot(HaveOccurred())
 
+				By("Clean resources after test")
 				err = idpService.gitlab.Destroy()
 				Expect(err).ToNot(HaveOccurred())
+
+				importService.Destroy()
```

Comment:
We need to check the destroy result with Expect.

Reply:
tnx! fixed